src/components/Agreements/ViewAgreement/ViewAgreement.js

```diff
@@ -73,6 +73,14 @@ class ViewAgreement extends React.Component {
       shouldRefresh,
       throwErrors: false,
     },
+    invoices: {
+      type: 'okapi',
+      path: 'invoice-storage/invoice-lines',
```

Comment:
Can you confirm that you've checked with their team about this path? They pushed back against us accessing `orders-storage` so I don't want to merge another bit of code that directly accesses their storage modules.

Reply:
Until now mod-invoice is "Not supported":
https://github.com/folio-org/mod-invoice/blob/master/src/main/java/org/folio/rest/impl/InvoicesImpl.java#L23